Makefile

```diff
@@ -186,12 +186,31 @@ run-test-multikueue-e2e-%: FORCE
 	@echo Running multikueue e2e for k8s ${K8S_VERSION}
 	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) ./hack/multikueue-e2e-test.sh
 
+.PHONY: install-shellcheck
+install-shellcheck: 
+	# removing previous shellcheck files if shellcheck give errors and warnings it wouldn't remove the downloaded files
+	rm -rf shellcheck-v0.9.0*
+	@echo "Installing ShellCheck"
+	curl -s https://api.github.com/repos/koalaman/shellcheck/releases/latest \
```

Comment:
I don't think we should have such an automation logic.
Should we just download a binary using https://github.com/koalaman/shellcheck/releases/download/v0.9.0/shellcheck-v0.9.0.darwin.x86_64.tar.xz?